docs/changehistory/NextVersion.md

```diff
@@ -8,17 +8,23 @@ Table of contents:
 - [Display system](#display-system)
   - [Dynamic schedule scripts](#dynamic-schedule-scripts)
   - [Hiliting models and subcategories](#hiliting-models-and-subcategories)
+- [Frontend category APIs](#frontend-category-apis)
   - [Improved appearance overrides for animated views](#improved-appearance-overrides-for-animated-views)
   - [Corrected material opacity](#corrected-material-opacity)
-- [Frontend category APIs](#frontend-category-apis)
 - [AppUi](#appui)
   - [Auto-hiding floating widgets](#auto-hiding-floating-widgets)
-  - [Tool Settings title](tool-settings-title)
+  - [Tool Settings title](#tool-settings-title)
 - [ElectronApp changes](#electronapp-changes)
 - [IModelHostOptions](#imodelhostoptions)
 - [Progress API for downloading changesets](#progress-api-for-downloading-changesets)
-- [Deprecations](#deprecations)
+- [RPC over IPC](#rpc-over-ipc)
 - [Presentation](#presentation)
+  - [Relationship properties](#relationship-properties)
+- [Tooling](#tooling)
```

Comment:
Let's call this out as webpack 5 specifically in the TOC, that way it doesn't get lost when people review it.

Reply:
```suggestion
- [Webpack 5](#webpack-5)
```